db/migrate/20211112213456_create_vaccines.rb

```diff
@@ -0,0 +1,10 @@
+class CreateVaccines < ActiveRecord::Migration[6.1]
```

Comment:
If I understand this project correctly, we could also include the manufacturer as an attribute on Vaccine. Chika tells me that we're only including manufacturer for covid-19 vaccines at this time, so there's no concern about multiple manufacturers.

Reply:
As above this can move to the mobile module dir.

RE: potentially adding a manufacturer column. I think it's safe to add manufacturer here for now as we control the timing of the update and could migrate data later if we needed a `has_many` relationship (and a second table).